src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/hmi_command_factory.cc

```diff
@@ -854,23 +827,7 @@ CommandCreator& HMICommandFactory::get_creator_factory(
                        .GetCreator<commands::VIGetVehicleDataResponseTemplate<
                            hmi_apis::FunctionID::VehicleInfo_GetMyKey> >();
     }
-#else
-    case hmi_apis::FunctionID::VehicleInfo_GetVehicleData: {
-      return hmi_apis::messageType::request == message_type
-                 ? factory.GetCreator<commands::VIGetVehicleDataRequest>()
-                 : factory.GetCreator<commands::VIGetVehicleDataResponse>();
-    }
 #endif  // #ifdef HMI_DBUS_API
```

Comment:
why VehicleInfo_GetMyKey and others in #ifdef HMI_DBUS_API not moved to the new plugin?

Reply:
@yang1070 this RPCs are applicable only for HMI_DBUS_API that is not supported by SDL right now and as I know will be removed within 5.0 ( same version as for this changes)

Reply:
@LuxoftAKutsan Sounds good to me.